### PR TITLE
WD-8531 - update text on /about/release-cycle

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Update text on /release-cycle according to copy doc

## QA

- [demo link](https://ubuntu-com-13512.demos.haus/about/release-cycle#ubuntu)
- [copy doc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the links and text are correct

## Issue / Card
[WD-8531](https://warthogs.atlassian.net/browse/WD-8531)

Fixes #

## Screenshots

[WD-8531]: https://warthogs.atlassian.net/browse/WD-8531?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/about/release-cycle.html [hunk 1]
```diff
@@ -61,15 +61,17 @@ <h3>Version numbers are YY.MM</h3>
...
+      </p>
 
       <p>Every six months between LTS versions, Canonical publishes an interim release of Ubuntu, with {{ releases.latest.short_version }} being the latest example. These are production-quality releases and are supported for 9 months, with sufficient time provided for users to update, but these releases do not receive the long-term commitment of LTS releases.</p>
     </div>
 
     {% include "/about/release_cycles/ubuntu-eol.html" %}
 
     <div class="u-fixed-width">
-      <p>ESM requires Ubuntu Pro subscription.</p>
+      <p>ESM phone and ticket support require Ubuntu Pro subscription.</p>
```
Comment (lorumic) [inline templates/about/release-cycle.html:74]: There's a missing "and" here:
```suggestion
      <p>ESM and phone and ticket support require Ubuntu Pro subscription.</p>
```

**Review by lorumic — COMMENTED** [review]
I guess the "Ceph" tab that appears in the copy doc is still a work-in-progress and out of the scope of this PR?

**Comment by lizzochek** [comment]
@lorumic accepted your change, please review again

**Review by lorumic — APPROVED** [review]
Thanks for the quick fix. Everything else looks good!

**Pull request merged**